services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/DefaultAttributeConverterProvider.java

```diff
@@ -88,6 +88,8 @@
 @ThreadSafe
 @Immutable
 public final class DefaultAttributeConverterProvider implements AttributeConverterProvider {
+    private static DefaultAttributeConverterProvider INSTANCE = getDefaultBuilder().build();
+
     private static final Logger log = Logger.loggerFor(DefaultAttributeConverterProvider.class);
 
     private final ConcurrentHashMap<EnhancedType<?>, AttributeConverter<?>> converterCache =
```

Comment:
Agree since this is created only at the time of creation the `ConcurrentHashMap` looks unecessary.
Will not modify this Class as part of DocumentAPI since this class is used by EnhancedClient , we can raise a separate PR for this.